Added blurbs for preview-5 content (#648)
dotnet/docs-aspire: docs/whats-new/preview-5.md

- HTTPs by default.
- Test project support. For more information, see [.NET Aspire project templates](../fundamentals/setup-tooling.md#net-aspire-project-templates).

## Service Discovery API changes

In preview 5, we've made changes to the service discovery API. The `UseServiceDiscovery` method was marked as obsolete and replaced with the `AddServiceDiscovery` method. The `AddServiceDiscovery` method is used to add service discovery to the application model. The `UseServiceDiscovery` method is still available but will be removed in preview 6. For more information, see [.NET Aspire service discovery](../service-discovery/overview.md).

> In addition to these breaking API changes, service discovery now support auto scheme selection. It's common to use HTTP while developing and testing a service locally and HTTPS when the service is deployed. Service discovery supports this by allowing for a priority list of URI schemes to be specified in the input string given to Service discovery. Service discovery attempts to resolve the services for the schemes in order and will stop after an endpoint is found. URI schemes are separated by a + character, for example: `"https+http://basket"`. Service discovery will first try to find HTTPS endpoints for the "basket" service and will then fall back to HTTP endpoints. If any HTTPS endpoint is found, Service Discovery will not include HTTP endpoints. For more information, see [Scheme selection when resolving HTTP(S) endpoints](/dotnet/core/extensions/service-discovery?tabs=dotnet-cli#scheme-selection-when-resolving-https-endpoints).

Reviewer: ```suggestion
In addition to these breaking API changes, service discovery now supports auto scheme selection. It's common to use HTTP while developing and testing a service locally and HTTPS when the service is deployed. Service discovery supports this by allowing for a priority list of URI schemes to be specified in the input string given to Service discovery. Service discovery attempts to resolve the services for the schemes in order and stops after an endpoint is found. URI schemes are separated by a + character, for example: `"https+http://basket"`. Service discovery first tries to find HTTPS endpoints for the "basket" service and then falls back to HTTP endpoints. If any HTTPS endpoint is found, Service Discovery doesn't include HTTP endpoints. For more information, see [Scheme selection when resolving HTTP(S) endpoints](/dotnet/core/extensions/service-discovery?tabs=dotnet-cli#scheme-selection-when-resolving-https-endpoints).
```
